xml/docu_styleguide.structure.xml

```diff
@@ -923,6 +923,128 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
   </para>
  </sect2>
 
+ <sect2 xml:id="sec-link-to-susecom">
+  <title>External links to &suse; documentation</title>
+  <para>
+   In some cases, it makes sense to provide a link which is independent
+   from any Service Pack (SP) number:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     Your linked information is independent from any SP
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     You cannot or do not need to pick a specific SP
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     You want to link to the most recent SP without checking which one it is
+    </para>
+   </listitem>
+   <listitem>
+   <para>
+     You want to support SEO where the most prominent SP is more important 
+     than previous, older SPs
+    </para>
+   </listitem>
+  </itemizedlist>
+  <para>
+   Make sure to follow this syntax:
+  </para>
+  <screen>&lt;documentation.suse.com/<replaceable>&lt;PRODUCT></replaceable>[/<replaceable>&lt;MAJOR_RELEASE></replaceable>]/<replaceable>&lt;DEEP_LINK>&gt;</replaceable></screen>
+  <para>
+   The placeholders mean the following:
+  </para>
+  <itemizedlist>
+    <listitem>
+    <para>
+     <replaceable>PRODUCT</replaceable>: the abbreviation of the product, like <quote>sle</quote> for &sle;,
+     <quote>sle-ha</quote> for &sle; Server High Availability Extension, etc.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     <replaceable>MAJOR_RELEASE</replaceable>: an optional major release like 12 or 15
```

Comment:
Maybe add an additional information what happens, when you don't provide a major release:

```suggestion
     <replaceable>MAJOR_RELEASE</replaceable>: an optional major release like 12 or 15. If you omit it,
     your link will be redirected to use the most recent release.
```